storage/client/tests.yml

```diff
@@ -0,0 +1,19 @@
+trigger: none
+
+jobs:
+    # When migrating change path to ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests-pre-sdk.yml
+    parameters:
+      # When migrating use ServiceDirectory
+      PomFilePath: './storage/client/pom.xml'
+      EnvVars:
+        AZURE_TEST_MODE: RECORD
```

Comment:
AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, AZURE_TENANT_ID

Are these able to get from the archetype-sdk-tests-pre-sdk.yml ?

Reply:
they come from variable groups / variable in the pipeline